x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/LifecycleSettings.java

```diff
@@ -128,4 +129,15 @@ public class LifecycleSettings {
         Setting.Property.Dynamic,
         Setting.Property.NodeScope
     );
+    /**
+     * The number of repeated failures allowed since the last successful SLM snapshot before a health warning is surfaced in the
+     * health API.
+     */
+    public static final Setting<Long> SLM_HEALTH_FAILED_SNAPSHOT_WARN_THRESHOLD_SETTING = Setting.longSetting(
```

Comment:
should we document this new setting?

Reply:
added in [ebc5e54](https://github.com/elastic/elasticsearch/pull/88523/commits/ebc5e546cdc88014064ca26c12ba45c3d41817e3)